server/authorization/router/auth.router.js

```diff
@@ -12,29 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import express from 'express';
+import Application from '../../base/application.js';
 import * as auth from '../controller/auth_controller';
 
-const router = express.router();
-
-router.post('/uuid', (request, response) => {
-  auth.generateUuid().then((authInfo) => {
-    response.send(authInfo);
-  }, (error) => {
-    response.sendStatus(400);
-  });
-});
+@Application.route('/uuid')
+export default class UuidRouter {
+  get(request, response) {
+    if (request.query.uuid) {
+      auth.getUuid(request.query.uuid).then((authInfo) => {
+        response.json(authInfo);
+      }, (error) => {
+        response.sendStatus(400);
```

Comment:
I think the 400 error is incorrect here.
The 400 error means "bad request" but it happens when the url syntax error.
So, we probably should use 403 error "Forbidden" instead of 400.